app/views/pregnancies/_pledge.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="modal pledge_modal">
```

Comment:
Just want to reiterate that this looks so, so, so, so good, and every CM I've shown it to has had heart palpitations from excitement.